modin/engines/dask/pandas_on_dask/frame/partition_manager.py

```diff
@@ -51,31 +75,31 @@ def deploy_func(df, apply_func, call_queue_df=None, call_queues_other=None, *oth
 
 
 class DaskFrameManager(BaseFrameManager):
-    """This class implements the interface in `BaseFrameManager`."""
+    """The class implements the interface in `BaseFrameManager`."""
 
-    # This object uses RayRemotePartition objects as the underlying store.
+    # This object uses PandasOnDaskFramePartition objects as the underlying store.
     _partition_class = PandasOnDaskFramePartition
     _column_partitions_class = PandasOnDaskFrameColumnPartition
     _row_partition_class = PandasOnDaskFrameRowPartition
 
     @classmethod
     def get_indices(cls, axis, partitions, index_func):
         """
-        This gets the internal indices stored in the partitions.
+        Get the internal indices stored in the partitions.
 
         Parameters
         ----------
-            axis : 0 or 1
-                This axis to extract the labels (0 - index, 1 - columns).
-            partitions : NumPy array
-                The array of partitions from which need to extract the labels.
-            index_func : callable
-                The function to be used to extract the function.
+        axis : 0 or 1
+            Axis to extract the labels over.
+        partitions : np.ndarray
+            The array of partitions from which need to extract the labels.
+        index_func : callable, default: None
```

Comment:
I don't see a default value for `index_func` in the method signature:
```suggestion
        index_func : callable
```